### PR TITLE
Add `wallet.dumpPrivKey` RPC

### PR DESCRIPTION
**Pull request by siradji**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
 RPC implementation #48 
#### Which issue(s) does this PR fixes?:

Fixes #

### CODE REVIEW

File: packages/jellyfish-api-core/__tests__/category/wallet.test.ts [hunk 1]
```diff
@@ -631,4 +631,21 @@ describe('masternode', () => {
...
     })
   })
+
+  describe('dumpPrivKey', () => {
+    it('should reveal private key of given address', async () => {
+      await waitForExpect(async () => {
+        const address = await client.wallet.getNewAddress()
+        const privateKey = await client.wallet.dumpPrivKey(address)
+
+        expect(typeof privateKey).toStrictEqual('string')
```
Comment (canonbrother) [inline packages/jellyfish-api-core/__tests__/category/wallet.test.ts:641]: its recommended to expect a exact output
refer https://github.com/DeFiCh/jellyfish/blob/35a5f074c1d1789f40733360baedf5aee2aa7534/packages/testcontainers/src/testkeys.ts

**Review by canonbrother — CHANGES_REQUESTED** [review]
need add the `dumpPrivKey` on *non masternode* test as well

File: packages/jellyfish-api-core/__tests__/category/wallet.test.ts (hunk 1 above)
Comment (siradji) [reply packages/jellyfish-api-core/__tests__/category/wallet.test.ts:641]: Those test keys are different from what i am getting. Is that normal?

**Comment by siradji** [comment]
### Behavior 
If i run test on the `dumprivkey` suite individually, it works well and pass but fails when i run test on  the`masternode` suite  or the whole test file. 


### fails at  

```

  describe('dumpPrivKey', () => {
    it('should reveal private key of given address', async () => {
      await waitForExpect(async () => {
        const address = await client.wallet.getNewAddress()
        const privateKey = await client.wallet.dumpPrivKey(address)

        expect(typeof privateKey).toStrictEqual('string')
      })
    })

    it('should throw and error when invalid DFI address is provided', async () => {
      await waitForExpect(async () => {
        const invalidAddress = 'invalidAddress'

        await expect(client.wallet.dumpPrivKey(invalidAddress)).rejects.toThrow('Invalid Defi address')
      })
    })
  })
```

### Error that i get

image

**Pull request closed without merging**